lib/plugins/ticker/ccxt.js

```diff
@@ -0,0 +1,57 @@
+const _ = require('lodash/fp')
+const axios = require('axios')
+const ccxt = require('ccxt')
+
+const BN = require('../../bn')
+const { buildMarket } = require('../common/ccxt')
+
+function ticker (fiatCode, cryptoCode, tickerName) {
+  const ticker = new ccxt[tickerName]({ timeout: 3000 })
+  if (fiatCode === 'EUR' || fiatCode === 'USD' || ticker.id === 'coinbase') {
+    return getCurrencyRates(ticker, fiatCode, cryptoCode)
+  }

```

Comment:
Shouldn't this be generic with the buildMarket thing?

Reply:
This is used only to skip Bitpay forex rates under those circumstances.